portal/src/components/Materials/Materials.component.js

```diff
@@ -54,20 +51,46 @@ export default {
       }
     },
     /**
-     * Select material
+     * Select material from collection
      * @param material - {Object}
+     * @param selectFor - string
      */
-    selectMaterial(material) {
-      let selected_materials = this.value.slice(0)
-
-      if (selected_materials.indexOf(material.external_id) === -1) {
-        selected_materials.push(material.external_id)
+    selectMaterial(material, selectFor) {
```

Comment:
selectFor hoeft niet als argument mee toch? Je kan ook `this.selectFor` gebruiken.

Reply:
✅